pkg/security/probe/process_resolver.go

```diff
@@ -121,6 +121,8 @@ type ProcessResolver struct {
 	pidCacheMap      *lib.Map
 	cacheSize        int64
 	opts             ProcessResolverOpts
+	hitsStats        map[string]*int64
+	missStats        *int64
```

Comment:
I'm curious why these are pointers.  I don't see specific handling of `nil` values in the patch, nor places they are set to nil.

Reply:
So I changed the missStats to a `int64` (I wasn't totally done with the PR when you reviewed it). As for the map, you can't take the address of a map element, and you need a pointer to use `atomic.SwapInt64()`.

Reply:
Ah, sorry -- I even checked whether it was a draft PR before reviewing!